src/VisualStudio/VisualBasic/Impl/Options/VisualBasicSettingsManagerOptionSerializer.vb

```diff
@@ -165,9 +167,26 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Options
                 Return FetchEnterKeyBehavior(optionKey, value)
             End If
 
+            If optionKey.Option Is CompletionOptions.TriggerOnDeletion Then
+                Return FetchTriggerOnDeletion(optionKey, value)
+            End If
+
             Return MyBase.TryFetch(optionKey, value)
         End Function
 
+        Private Function FetchTriggerOnDeletion(optionKey As OptionKey, ByRef value As Object) As Boolean
+            If MyBase.TryFetch(optionKey, value) Then
+                If value Is Nothing Then
+                    ' The default behavior for VB is to trigger completion on deletion.
+                    value = CType(True, Boolean?)
```

Comment:
So the problem is as follows:

1. Our per language options only allow us to set a default across all languages.  
2. We don't want the default for this option to be either true or false.  Each of those is wrong for one of our two primary languages.
3. So this is actually a tri-state.  'null' means 'language default'.  'true' and 'false' means "the user chose these explicit values.

We use this pattern for all options that differ in their defaults between C# and VB.  It allows us to have a 'default' that still gets the right legacy/historical behavior for these two languages.